chart/templates/grafana-db-sec.yaml

```diff
@@ -1,5 +1,5 @@
 {{- $grafana := index .Values "kube-prometheus-stack" "grafana" -}}
```

Comment:
Can we just remove this whole file? This file was useful when using timescale as grafana database, however we deprecated this mode and thus this could be removed.

Reply:
👍